### DISCUSSION TITLE
Forms: Converting strings to numbers simplified

### DISCUSSION THREAD

**Discussion by caseywalker** [nss-evening-cohort-14/evening-client-side#295] [Look at what I learned!]
We've discussed a few ways to convert numbers to strings, but I'm not sure if we've discussed this way. This helped me as I need my values to be numbers, and not strings. Changing from e.target.value to e.target.valueAsNumber will store the value as a number. 

 ``` 
const handleNumberInput = (e) => {
    setSaving((prevState) => ({
      ...prevState,
      [e.target.name]: e.target.valueAsNumber
    }));
  };

<input onChange={handleNumberInput} />
```

**Comment by Mitchell-Crumbley** [comment]
game changer. Thank you so much
